Source/WebCore/dom/DocumentMarkerController.cpp

```diff
@@ -711,6 +726,17 @@ void DocumentMarkerController::dismissMarkers(OptionSet<DocumentMarker::Type> ty
         m_fadeAnimationTimer.startRepeating(1_s / markerFadeAnimationFrameRate);
 }
 
+void DocumentMarkerController::unifiedTextReplacementAnimationTimerFired()
+{
+#if ENABLE(UNIFIED_TEXT_REPLACEMENT)
+    forEachOfTypes({ DocumentMarker::Type::UnifiedTextReplacement }, [&] (Node& node, RenderedDocumentMarker&) {
```

Comment:
It looks like we're not using any captured value, so we can just drop `&`? Also small nit, style guide says no space between capture list and arg list.
```suggestion
    forEachOfTypes({ DocumentMarker::Type::UnifiedTextReplacement }, [](Node& node, RenderedDocumentMarker&) {
```